decidim-core/lib/tasks/upgrade/clean.rake

```diff
@@ -9,9 +9,16 @@ namespace :decidim do
         :"decidim:upgrade:clean:notifications",
         :"decidim:upgrade:clean:follows",
         :"decidim:upgrade:clean:categories",
-        :"decidim:upgrade:clean:action_logs"
+        :"decidim:upgrade:clean:action_logs",
+        :"decidim:upgrade:clean:clean_deleted_users"
       ]
 
+      desc "Remove data from deleted users"
+      task clean_deleted_users: :environment do
+        logger.info("=== Removing extra data from deleted users")
+        Decidim::User.where.not(deleted_at: nil).update_all(personal_url: "", about: "") # rubocop:disable Rails/SkipsModelValidations
```

Comment:
Can you add an spec on `decidim-core/spec/tasks/upgrade/` 🙏🏽 ? Just checking for a deleted user and for a non deleted user would be enough

Reply:
done in e9d75eee2d